features/list-streams.feature

```diff
@@ -0,0 +1,10 @@
+Feature: List-streams
+    As a User
+    I want to see a list of my data streams
+    So that i can analyze them
+
+Scenario: Successful List
```

Comment:
The indentation is incorrect, check this (page)[https://cucumber.io/docs/gherkin/reference/] for reference.

Also, I don't know if hyphens are supported in Feature names. Maybe we could change it to `List streams`